src/rebuild/scan.c

```diff
@@ -205,9 +206,12 @@ rebuild_objects_send(struct rebuild_root *root, unsigned int tgt_id,
 	D_DEBUG(DB_REBUILD, "send rebuild objects "DF_UUID" to tgt %d"
 		" cnt %d\n", DP_UUID(rpt->rt_pool_uuid), tgt_id, arg->count);
 
+	ABT_rwlock_rdlock(rpt->rt_pool->sp_lock);
```

Comment:
I'm bit confused, rebuild leader calls rebuild_pool_map_get() to hold a refcount of pool_map, but all the rebuild scan code actually uses 'rt_pool->sp_map' instead of the pool_map held by rebuild_pool_map_get(), so what's the point of this rebuild_pool_map_get()?

Also I noticed the locking is missed in some other places, should we do a cleanup to fix all the potential problems?

Reply:
oh, that is for update placement map, but indeed, that code has been obsolete after we do that inside pool map update, I will remove that.

Ok, sure I will do further lock up.